quickstarts/ocs-install-tour-quickstart.yaml

```
@@ -0,0 +1,70 @@
+  durationMinutes: 5
+  icon: data:image/svg+xml;base64,PHN2ZyBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCAxMDAgMTAwIiBoZWlnaHQ9IjEwMCIgdmlld0JveD0iMCAwIDEwMCAxMDAiIHdpZHRoPSIxMDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTY2LjcgNTUuOGM2LjYgMCAxNi4xLTEuNCAxNi4xLTkuMiAwLS42IDAtMS4yLS4yLTEuOGwtMy45LTE3Yy0uOS0zLjctMS43LTUuNC04LjMtOC43LTUuMS0yLjYtMTYuMi02LjktMTkuNS02LjktMy4xIDAtNCA0LTcuNiA0LTMuNSAwLTYuMS0yLjktOS40LTIuOS0zLjIgMC01LjIgMi4xLTYuOCA2LjYgMCAwLTQuNCAxMi41LTUgMTQuMy0uMS4zLS4xLjctLjEgMSAuMSA0LjcgMTkuMiAyMC42IDQ0LjcgMjAuNm0xNy4xLTZjLjkgNC4zLjkgNC44LjkgNS4zIDAgNy40LTguMyAxMS40LTE5LjEgMTEuNC0yNC42IDAtNDYuMS0xNC40LTQ2LjEtMjMuOSAwLTEuMy4zLTIuNi44LTMuOS04LjkuNS0yMC4zIDIuMS0yMC4zIDEyLjIgMCAxNi41IDM5LjIgMzYuOSA3MC4yIDM2LjkgMjMuOCAwIDI5LjgtMTAuNyAyOS44LTE5LjIgMC02LjctNS44LTE0LjMtMTYuMi0xOC44IiBmaWxsPSIjZWQxYzI0Ii8+PHBhdGggZD0ibTgzLjggNDkuOGMuOSA0LjMuOSA0LjguOSA1LjMgMCA3LjQtOC4zIDExLjQtMTkuMSAxMS40LTI0LjYgMC00Ni4xLTE0LjQtNDYuMS0yMy45IDAtMS4zLjMtMi42LjgtMy45bDEuOS00LjhjLS4xLjMtLjEuNy0uMSAxIDAgNC44IDE5LjEgMjAuNyA0NC43IDIwLjcgNi42IDAgMTYuMS0xLjQgMTYuMS05LjIgMC0uNiAwLTEuMi0uMi0xLjh6IiBmaWxsPSIjMDEwMTAxIi8+PC9zdmc+
+  description: Install the OpenShift Container Storage (OCS) operator and create a storage cluster.
+  introduction: Red Hat OpenShift Container Storage is persistent software-defined storage integrated with and optimized for Red Hat OpenShift Container Platform. Dynamic, stateful, and highly available container-native storage can be provisioned and de-provisioned on demand as an integral part of the OpenShift administrator console.
+  tasks:
+  - title: Install the OpenShift Container Storage operator
+    description: |- 
+      The OperatorHub is where you can find a catalog of available Operators to install on your cluster.
+      
+      To install OpenShift Container Storage, follow these steps:
+
+      1. In the navigation menu, select **Operators > OperatorHub**
+      2. In the search bar, enter  **OpenShift container storage**
+      3. Click the **OpenShift Container Storage** tile to initiate the operator installation on your cluster.
+      4. In the panel that appears, click **Install.** 
+      5. On the operator installation page, fill out the Operator Subscription form. 
+      6. Click **Install.**
+    review:
+      instructions: |-
+        ####  Verify that the OpenShift Container Storage operator is installed:
+
+        In the navigation menu, click **Operators** > **Installed Operators**. 
+        
+        Does the **Status** column for **OpenShift Container Storage** show **Succeeded**?
+      failedTaskHelp: This task isn’t verified yet. Try the task again.
+    summary:
+      success:  Great work! You installed the OpenShift Container Storage operator.
+      failed: Try the steps again.
+  - title: Create a storage cluster
+    description: |-
+      A storage cluster includes a Ceph cluster, Multi cloud gateway and all the required storage and commute resources .
```

```suggestion
      A storage cluster includes a Ceph cluster, Multi cloud gateway, and all the required storage and commute resources.
``` 

There was just a missing comma and extra space here, so I added that above in the edit.

---

done